11LessonEleven/in-class-project/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>Intel</title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Lesson 11 in class</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Lora" rel="stylesheet">
 </head>
-<body>
 
+<body>
+    <header class="header">
+        <h1> Welcome to our site</h1>
+    </header>
+    <main class="main" class="clearfix">
+        <div class="left">
+            <div class="colorblue">
+            </div>
+            <h1>Title</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        </div>
+        <div class="center">
+            <div class="colororange">
+            </div>
+            <h1>Title</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        </div>
+        <div class="right">
+            <div class="colorgreen">
+            </div>
+            <h1>Title</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        </div>
+    </main>
 </body>
+
 </html>
```

Comment:
Please fix the layout here and I can bump your grade up.

![screen shot 2016-09-13 at 5 42 22 pm](https://cloud.githubusercontent.com/assets/12698279/18494073/b56ca7a0-79d9-11e6-8c35-abe98b8bf3c5.png)